profile_library/jbl/manufacturer.json

```
@@ -0,0 +1,4 @@
+{
+  "name": "JBL",
+  "aliases": []
```

```suggestion
  "aliases": ["HARMAN International Industries"]
```

`HARMAN International Industries` is the actual manufacturer as returned in device info from HA, so this must be added as an alias to make sure auto discovery is working correctly.